src/protocol/commands/_vFile_open.rs

```diff
@@ -0,0 +1,28 @@
+use super::prelude::*;
+
+#[derive(Debug)]
+pub struct vFileOpen<'a> {
+    pub filename: &'a [u8],
+    pub flags: u64,
```

Comment:
This should be an enum corresponding to:

https://sourceware.org/gdb/onlinedocs/gdb/Open-Flags.html#Open-Flags

Reply:
I'd crib the implementation from https://github.com/luser/rust-gdb-remote-protocol/blob/master/src/fs.rs#L90-L110

the `bitflags` dependency is no_std, and pretty light, so I wouldn't mind including it.